src/promptflow/promptflow/contracts/trace.py

```diff
@@ -8,13 +8,37 @@
 
 
 class TraceType(str, Enum):
+    """An enumeration class to represent different types of traces."""
+
     LLM = "LLM"
     TOOL = "Tool"
     LANGCHAIN = "LangChain"
 
 
 @dataclass
 class Trace:
+    """A dataclass that represents a trace of a program execution.
+
+    :param name: The name of the trace.
+    :type name: str
+    :param type: The type of the trace.
+    :type type: TraceType
```

Comment:
`TraceType` -> `~promptflow.xxx.TraceType`

Reply:
Added.

BTW, what's the purpose of adding `~`?

Reply:
See the pic I replied in above comments